app/assets/javascripts/index/element.js

```diff
@@ -171,8 +171,8 @@
       .addClass("bg-body-tertiary");
 
     if (icon && OSM.WIKIMEDIA_COMMONS_URL) {
-      let src = OSM.WIKIMEDIA_COMMONS_URL + "Special:Redirect/file/" + encodeURIComponent(icon);
-      if (!icon.endsWith(".svg")) src += "?width=128";
+      let src = OSM.WIKIMEDIA_COMMONS_URL + "Special:Redirect/file/" + encodeURIComponent(icon) + "?mobileaction=toggle_view_desktop";
```

Comment:
In cursory testing, this doesn’t cause subsequent user-initiated page loads on Commons to switch back to desktop view, which is good. Still a bit counterintuitive in this code, so a comment explaining the parameter would be a good idea.

Reply:
A longer comment can be added to the commit message, to avoid making whoever finds this change looking up its pull request.